.github/buildomat/jobs/deploy.sh

```diff
@@ -166,7 +210,34 @@ ZPOOL_VDEV_DIR=/scratch ptime -m pfexec ./tools/create_virtual_hardware.sh
 tar xf out/omicron-sled-agent.tar pkg/config-rss.toml
 SILO_NAME="$(sed -n 's/silo_name = "\(.*\)"/\1/p' pkg/config-rss.toml)"
 EXTERNAL_DNS_DOMAIN="$(sed -n 's/external_dns_zone_name = "\(.*\)"/\1/p' pkg/config-rss.toml)"
-rm -f pkg/config-rss.toml
+
+# Substitute addresses from the external network range into the RSS config.
+sed -i~ "
```

Comment:
This is some impressive `sed`'ing 😅 but seems like this script could stand to not be bash soon (not a blocker for this PR to be clear)

Reply:
I didn't do that because I'm also using `PXA_START` - `PXE_END` to configure the IP pool through the API in the end-to-end tests bootstrap. I'll adjust that and simplify this script a bit.